src/trigger.py

```diff
@@ -82,6 +82,7 @@ def _run_trigger(self, build_config, force, timeout, trees):
             'kind': 'checkout',
             'data': {
                 'kernel_revision': revision,
+                'tip_of_branch': True,
```

Comment:
This shouldn't be set statically. We should keep it dynamic.

Reply:
In trigger it is by definition tip of tree:
```
        head_commit = kernelci.build.get_branch_head(build_config)
```
so it is expected that trigger.py generates only branch head which is same meaning as tip of branch.